pmd-plsql/src/test/java/net/sourceforge/pmd/lang/plsql/ast/SelectIntoStatementTest.java

```diff
@@ -17,7 +19,12 @@ public void testParsingComplex() {
 
     @Test
```

Comment:
You can use `@Test(expect = ParseException.class)` - then you don't need to write the try-catch manually.

We should add another test case with the fixed file "SelectIntoStatementExample1.pls", otherwise we don't verify anymore, that we can parse "SELECT INTO" statements...

Reply:
Hmm... Isn't PMD using JUnit 5? I'm a JUnit noob, but the docs I found say that the expect syntax you show was valid only up to JUnit 4, but not in JUnit 5.  And in JUnit 5 the recommended way seems to use an Executable functional interface (see eg. https://www.baeldung.com/junit-assert-exception). But functional interfaces are not allowed in PMD unfortunately.
That's why I chose this way.

Reply:
We mostly use junit 4 for now, and will transition to junit 5 + java 8 in pmd 7. In Junit 4 the recommended way is to use an [`@Rule ExpectedException`](https://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html) field. Please use either the `@Test` syntax or ExpectedException, as either way these will have to be ported to junit 5's assertThrows in pmd 7.